docs/types.rst

```diff
@@ -96,6 +96,16 @@ In Solidity, division rounds towards zero. This mean that ``int256(-5) / int256(
 Note that in contrast, division on :ref:`literals<rational_literals>` results in fractional values
 of arbitrary precision.
 
+The modulus operator ``%`` yields the remainder from the division of the left operand by the right operand.
```

Comment:
I think we should call this `modulo operator` ( https://en.wikipedia.org/wiki/Modulo_operation ) - please also correct in other places.

Reply:
Also perhaps `after the division` and not `from`